releases.yaml

```diff
@@ -29,16 +31,16 @@ previous_previous_lts:
 
 checksums:
   desktop:
-    "20.10": "3ef833828009fb69d5c584f3701d6946f89fa304757b7947e792f9491caa270e *ubuntu-20.10-desktop-amd64.iso"
+    "21.04": "3ef833828009fb69d5c584f3701d6946f89fa304757b7947e792f9491caa270e *ubuntu-20.10-desktop-amd64.iso"
```

Comment:
It's it with leaving these as XXXX to make it clear they need to be updated?

Reply:
@anthonydillon happy to, I already have an issue for it - https://github.com/canonical-web-and-design/ubuntu.com/issues/9565